base/src/main/java/io/spine/code/proto/ref/InPackage.java

```diff
@@ -30,7 +30,7 @@
 /**
  * Reference to all message types in a proto package.
  */
-final class InPackage extends AbstractTypeRef {
+public final class InPackage extends AbstractTypeRef {
```

Comment:
You don't have to open this class, if you build in the knowledge about the short reference expansion into the base interface.